gui/index2.html

```diff
@@ -34,23 +41,15 @@
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
 	<script src="http://pixijs.download/release/pixi.min.js"></script>
 
-	<!-- blockly imports -->
-	<script src="./static/js/vendor/blockly/blockly_compressed.js"></script>
-	<script src="./static/js/vendor/blockly/javascript_compressed.js"></script>
-	<script src="./static/js/vendor/blockly/python_compressed.js"></script>
-	<script src="./static/js/vendor/blockly/blocks_compressed.js"></script>
-	<script src="./static/js/vendor/blockly/en.js"></script>
-	<script src="./static/js/vendor/prettify/prettify.js"></script>
-
 	<!-- IMPORT CUSTOM BLOCKS -->
 	<script type="text/javascript"
 			src="./static/js/blocks/custom_blocks.js"></script>
 	<script type="text/javascript"
 			src="./static/js/blocks/custom_json.js"></script>
-
+	<script src="./static/js/vendor/prettify/prettify.js"></script>
 	<!-- manual scripts -->
 	<script src="./static/build/app.bundle.js"></script>
-	<script src="./static/js/blockly.js"></script>
+	<!--<script src="./static/js/blockly.js"></script>-->
```

Comment:
Please provide a comment as to why you are commenting this out, if it is temporary. If you are planning on getting rid of this altogether just delete the line.